packages/api/src/types.ts

```diff
@@ -48,117 +49,118 @@ export * from '@polkadot/api/types';
 export type ApiTypes = 'promise' | 'rxjs';
 
 export interface ApiOptions extends Pick<ApiOptionsBase, Exclude<keyof ApiOptionsBase, 'provider'>> {
-    /**
-     * provider implement ProviderInterface or string url for WsProvider.
-     * If not specified, it will default to connecting to the
-     * localhost with the default port, i.e. `ws://127.0.0.1:9944`
-     */
-    provider?: ProviderInterface | string;
-    plugins?: IPlugin[];
-    /**
-     * timeout for Api.create
-     * default 10000 ms, 0 indicates no limit
-     */
-    timeout?: number;
+  /**
+   * provider implement ProviderInterface or string url for WsProvider.
+   * If not specified, it will default to connecting to the
+   * localhost with the default port, i.e. `ws://127.0.0.1:9944`
+   */
+  provider?: ProviderInterface | string;
+  plugins?: IPlugin[];
+  /**
+   * timeout for Api.create
+   * default 10000 ms, 0 indicates no limit
+   */
+  timeout?: number;
 }
 
 export interface IPlugin {
-    injectName?: string;
-    sdkClass?: Constructor<any>;
-    sdkRxClass?: Constructor<any>;
-    types?: RegistryTypes;
-    derives?: DeriveCustom;
+  injectName?: string;
+  sdkClass?: Constructor<any>;
+  sdkRxClass?: Constructor<any>;
+  types?: RegistryTypes;
+  derives?: DeriveCustom;
 }
 
 export interface SignerOptions extends SignerOptionsBase {
-    doughnut?: DoughnutValue;
-    feeExchange?: FeeExchangeValue;
+  // doughnut?: DoughnutValue;
```

Comment:
Can we remove `feeExchange` here?
Leave doughnut though

Reply:
yes..will do